torch/_export/converter.py

```diff
@@ -195,6 +195,13 @@ def get_op_overload(node: torch._C.Node):
     try:
         op_overload_mod = getattr(torch.ops, ns)
         op_overload_packet = getattr(op_overload_mod, op_name)
+
+        # To match the behavior of direct export, we replace specialized operator
+        # with its symbolic counterpart if exists.
+        op_overload_packet = getattr(
+            op_overload_mod, f"sym_{op_name}", op_overload_packet
```

Comment:
I think we should just hardcode a map of ops to sym-ops, just because these sym-ops are special